compiler/src/iree/compiler/Codegen/SPIRV/test/lowering_matmul_promotion.mlir

```diff
@@ -51,34 +51,49 @@ hal.executable @matmul_f32_128x256x64 {
   }
 }
 
-//       CHECK-DAG: #[[MAP:.+]] = affine_map<(d0) -> ((d0 floordiv 16) mod 2)>
-//       CHECK-DAG: #[[TRANSLATION:.+]] = #iree_codegen.translation_info<SPIRVMatmulPromoteVectorize pipeline_depth = 2>
+//       CHECK-DAG: #[[MAP:.+]] = affine_map<(d0) -> ((d0 floordiv 16) mod 3)>
+//       CHECK-DAG: #[[TRANSLATION:.+]] = #iree_codegen.translation_info<SPIRVMatmulPromoteVectorize pipeline_depth = 2 store_stage = 0>
 //           CHECK: hal.executable.export public @matmul_f32_128x256x64
 //      CHECK-SAME:   translation_info = #[[TRANSLATION]]
 //      CHECK-SAME:   workgroup_size = [16 : index, 8 : index, 1 : index]
 //           CHECK: func.func @matmul_f32_128x256x64()
 //           CHECK:   %[[CST0:.+]] = arith.constant 0.000000e+00 : f32
-//           CHECK:   memref.alloc() : memref<2x64x20xf32, 3>
-//           CHECK:   memref.alloc() : memref<2x16x68xf32, 3>
+//           CHECK:   memref.alloc() : memref<3x64x20xf32, 3>
+//           CHECK:   memref.alloc() : memref<3x16x68xf32, 3>
+//   CHECK-COUNT-8:   vector.transfer_write %{{.+}}, %{{.+}} {in_bounds = [true]} : vector<4xf32>, memref<128x256xf32>
+//   CHECK-COUNT-8:   vector.transfer_read %{{.+}}, %[[CST0]] {in_bounds = [true]} : memref<128x256xf32>, vector<4xf32>
```

Comment:
Hmm, I expect the `vector.transfer_write` generated from `linalg.fill` should be forwarded to the `vector.transfer_read` here.. Do you know why that's not happening? Would be nice to figure out why. Not exactly for this pull request, but would be nice to at least put a TODO here.